test/config.spec.ts

```diff
@@ -97,6 +97,14 @@ describe('Config', () => {
     expect(() => new Config({caseFirst: 'abcd'})).toThrowError('Expected one of \'upper\' \'lower\' \'false\' for config parameter: caseFirst')
   })
 
+  it('validation: context', () => {
+    expect(() => new Config({context: 'ABCD'})).toThrowError('Expected value of type: object for config parameter: context')
```

Comment:
Do you think we should validate the type of the `context` parameter? In my opinion, any type should be allowed.